examples/ex9.cpp

```diff
@@ -345,7 +367,7 @@ int main(int argc, char *argv[])
    // 8. Define the time-dependent evolution operator describing the ODE
    //    right-hand side, and perform time-integration (looping over the time
    //    iterations, ti, with a time-step dt).
-   FE_Evolution adv(m.SpMat(), k.SpMat(), b, fes);
+   FE_Evolution adv(m, k, b, fes, pa);
```

Comment:
The `pa` parameter can be obtained from the `BilinearForm`s, no need to pass it separately.

Reply:
Actually, `fes` can also be obtained from the `BilinearForm`s.